src/services/users/users.ts

```diff
@@ -22,6 +22,8 @@ export default class UserSdk {
         }
         return this.axiosClientHeaderless.post(`/users/auth/login`, data).then((res: any) => {
             this.logger.debug(res);
+        }).catch(error => {
```

Comment:
We shouldnt have a catch here. Also why are we modifying `users.ts` code in this PR? Unintentional? If so, revert these changes.

Reply:
i didnt change user.ts it couldnt of been the branch was behind and i didnt update the current branch

Reply:
all the fixes is in the user branch of the sdk once we merge it the problem will go away

Reply:
I think youll get a merge conflict here if you don't remove the change to this code that is on this branch. It's highlighted green because it's a change that was made on this branch.